crowdsec-docs/docs/intro.mdx

```diff
@@ -38,12 +47,39 @@ In addition to the core "detect and react" mechanism, CrowdSec is committed to s
 ## Architecture
 
 
+<!-- @kka : update schema:
+    - align text
+    - rename data sources to log sources
+    - move log sources out of the SE box
+-->
 <div style={{display: 'flex'}}>
     <div style={{textAlign: 'center', flex: '1'}}>
-    <img src={useBaseUrl('/img/crowdsec_architecture.png')}></img>
+    <img src={useBaseUrl('/img/simplified_SE_underthehood.png')}></img>
     </div>
 </div>
 
+Under the hood, the Security Engine has various components:
+ 
+ - The Log Processor is in charge of detection: it analyzes logs from [various data sources](/docs/data_sources/intro) or [HTTP requests](/appsec/intro.md) from web servers.
+ - The [WAF](/appsec/intro.md) feature is part of the Log Processor and filters HTTP Requests from the compatible web servers. 
+ - The [Local API](/local_api/intro.md) acts as a middle man between the [Log Processors](/docs/data_sources/intro) and the [Remediation Components](/u/bouncers/intro) which are in charge of enforcing decisions.
```

Comment:
That's it ?
don't we mention it's also interracting with CrowdSec network for crowd-sourced aspect and SaaS functions like the CrowdSEc Console